src/main/java/calculator/Operator.java

```diff
@@ -4,47 +4,31 @@
 import java.util.function.IntBinaryOperator;
 
 public enum Operator {
-    ADD("+") {
-        @Override
-        public IntBinaryOperator execute() {
-            return Integer::sum;
-        }
-    },
-    SUB("-") {
-        @Override
-        public IntBinaryOperator execute() {
-            return (a, b) -> a - b;
-        }
-    },
-    MUL("*") {
-        @Override
-        public IntBinaryOperator execute() {
-            return (a, b) -> a * b;
-        }
-    },
-    DIV("/") {
-        @Override
-        public IntBinaryOperator execute() {
-            return (a, b) -> a / b;
-        }
-    };
+    ADD("+", Integer::sum),
+    SUB("-", (left, right) -> left - right),
+    MUL("*", (left, right) -> left * right),
+    DIV("/", (left, right) -> left / right);
 
-    private String operator;
+    private String description;
+    private IntBinaryOperator operator;
 
-    Operator(String operator) {
+    Operator(String description, IntBinaryOperator operator) {
+        this.description = description;
         this.operator = operator;
     }
 
-    public String getOperator() {
-        return operator;
+    public String getDescription() {
+        return description;
     }
 
     public static Operator valueOfString(String str) {
         return Arrays.stream(Operator.values())
-                .filter(i -> str.equals(i.getOperator()))
+                .filter(i -> str.equals(i.getDescription()))
                 .findFirst()
```

Comment:
같은 클래스라면 필드에 바로 접근이 가능합니다 :) 

```suggestion
                .filter(i -> str.equals(i.description))
                .findFirst()
```

의미 없는 getter까지 제거 가능하겠네요!